pkg/probe/bgp_neighbors_test.go

```diff
@@ -0,0 +1,48 @@
+
+package probe
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/testutil"
+)
+
+func TestBGPNeighborsIPv4(t *testing.T) {
+	c := newFakeClient()
+	c.prepare("api/v2/monitor/router/bgp/neighbors", "testdata/bpg-neighbors-v4.jsonnet")
+	r := prometheus.NewPedanticRegistry()
+	if !testProbe(probeBGPNeighborsIPv4, c, r) {
+		t.Errorf("probeBGPNeighborsIPv4() returned non-success")
+	}
+
+	em := `
+	# HELP fortigate_bgp_neighbor_ipv4_info Confiured bgp neighbor over ipv4
```

Comment:
Why are the test lines differently indented?

Reply:
Fixed, this was missed by my editor